pkg/graph/schema.resolvers.go

```diff
@@ -1849,6 +1849,9 @@ func (r *systemIntakeResolver) Requester(ctx context.Context, obj *models.System
 		return nil, err
 	}
 
+	if user == nil {
+		return nil, nil
```

Comment:
is it ok to return `nil, nil` here? should it instead be some error for user not found?

Reply:
No, you're right: [5634d23](https://github.com/CMS-Enterprise/easi-app/pull/2706/commits/5634d234bde63994f13758434e879c19778acfbe)